ValveResourceFormat/CompiledShader/Enums/TextureFormat.cs

```diff
@@ -2,10 +2,67 @@ namespace ValveResourceFormat.CompiledShader;
 
 public enum TextureFormat
 {
-    Uncompressed = 0,
-    HDR_Something = 15,
+    UNKNOWN = -1,
+    RGBA8888 = 0,
+    ABGR8888 = 1,
+    RGB888 = 2,
+    BGR888 = 3,
+    RGB565 = 4,
+    I8 = 5,
+    IA88 = 6,
+    A8 = 7,
+    ARGB8888 = 8,
+    BGRA8888 = 9,
+    BGRX8888 = 10,
+    BGR565 = 11,
+    BGRA4444 = 12,
+    UV88 = 13,
+    UVWQ8888 = 14,
+    RGBA16161616F = 15,
+    RGBA16161616 = 16,
+    R32F = 17,
+    RGB323232F = 18,
+    RGBA32323232F = 19,
     DXT1 = 20,
+    DXT1_ONEBITALPHA = 21,
+    DXT3 = 22,
     DXT5 = 23,
-    Unkn27 = 27,
-    BC7 = 86,
+    ATI1N = 24,
+    ATI2N = 25,
+    BC6H = 26,
+    BC7 = 27,
+    DXT5_NM = 28,
+    R8G8B8_ETC2 = 29,
```

Comment:
Seems like this enum is only valid for Dota 2.

In Alyx/Sbox/Aperture Desk BC7 has value 86.

Reply:
Older Dota 2 shaders (pcgl v64) don't seem to use this enum either. Therefore I'll make it print these values only with version 66 files.

Should be worth looking up and including the < v66 enum.

Reply:
How convenient, I was able to retrieve the older enum from https://asset.party/api/Sandbox.ImageFormat.